src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
-// write code here
+const tbody = document.querySelector('tbody');
+const theadList = [...document.querySelector('thead tr').children];
+const rows = [...tbody.rows];
+
+document.body.addEventListener('click', e => {
```

Comment:
You can add eventListener not for the whole body, but only for table, or even its header